src/Normalizer/FieldNormalizer.php

```diff
@@ -96,7 +96,7 @@ public function denormalize($data, $class, $format = NULL, array $context = arra
    * @return array
    *   The array of normalized field items.
    */
-  protected function normalizeFieldItems($field, $format, $context) {
+  protected function normalizeFieldItems(FieldItemListInterface $field, string $format, array $context) {
```

Comment:
Type hinting on primitive datatypes is not supported in PHP 5
http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration